r2c.bandit:
  violations:
    30fad05838897b7ff4f15ffc8e194660:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: 'proc = subprocess.Popen(

        cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding="utf8"

        )

        stdout, stderr = proc.communicate()'
      tool_id: r2c.bandit
    5a4c9f7287dcc63f65ede98ac81ee151:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: 'proc = subprocess.Popen(

        cmd, env=env, shell=True, stdout=subprocess.PIPE, encoding="utf8"

        )

        res, _ = proc.communicate()'
      tool_id: r2c.bandit
    94f271aa5818b07a58371ffb938b83cc:
      check_id: B110
      message: Try, Except, Pass detected.
      path: tests/commands/test_check.py
      severity: 0
      syntactic_context: 'assert False

        except Exception:

        pass'
      tool_id: r2c.bandit
    a818cd9a70b1ccca74b3687b7f4355c5:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: bento/extra/python_tool.py
      severity: 2
      syntactic_context: 'wrapped,

        shell=True,

        cwd=str(self.base_path),

        encoding="utf8",

        executable="/bin/bash",

        env=env,

        stdout=subprocess.PIPE,

        stderr=subprocess.PIPE,

        )

        stdout, stderr = v.communicate()

        logging.debug(f"{self.tool_id()}: stderr[:4000]:\n" + stderr[0:4000])'
      tool_id: r2c.bandit
    cb09ff7333a5a433b189c697bfa69116:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: 'env=env,

        shell=True,

        stdout=subprocess.PIPE,

        stderr=subprocess.PIPE,

        encoding="utf8",

        )

        stdout, stderr = proc.communicate()

        assert not stdout'
      tool_id: r2c.bandit
    f8ea89cbc1237d72dcb7ac244658cee3:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: '# NOTE: This is a deliberate demonstration of a shell injection

        proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, encoding="utf8")

        res, _ = proc.communicate()'
      tool_id: r2c.bandit
r2c.eslint:
  violations:
    c6e0f687a580ef5e681dc3b4a7ed7fce:
      check_id: no-undef
      message: '''x'' is not defined.'
      path: tests/acceptance/bad.js
      severity: 2
      syntactic_context: x = "potato";
      tool_id: r2c.eslint
r2c.flake8:
  violations: {}
